src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -115,6 +115,13 @@ object ScalafixPlugin extends AutoPlugin {
 
   import autoImport._
 
+  private val postScalafixHooks: SettingKey[Seq[Seq[File] => Unit]] =
```

Comment:
I guess we want to keep this hidden for now? I have used plural so that we can safely advertise it to other plugins in the feature without breaking sbt-scalafmt